src/BaseVault.sol

```diff
@@ -780,6 +785,35 @@ abstract contract BaseVault is IVault, ERC20PermitUpgradeable, AccessControlUpgr
         _getVaultStorage().totalAssets = totalBaseBalance;
         // solhint-disable-next-line not-rely-on-time
         emit ProcessAccounting(block.timestamp, totalBaseBalance);
+
+        address feeModule_ = feeModule();
+        if (feeModule_ != address(0)) {
+            IFeeModule(feeModule_).chargePerformanceFee();
```

Comment:
I suggest we make a simple naming change that just confines more generality.

We can use the IHooks name instead of IFeeModule

and this particular function would be hooks.afterProcessAccounting()

This follows the Uniswap V4 naming pattern.

https://github.com/Uniswap/v4-core/blob/main/src/PoolManager.sol#L178C36-L178C45

This way it follows a more general pattern and we can potentially have other functionality there without making the name unfitting.